lib/broken_record/slack_notifier.rb

```diff
@@ -1,27 +1,52 @@
 require 'net/http'
 module BrokenRecord
   class SlackNotifier
-    def initialize(options)
-      @options = BrokenRecord::Config.slack_options.merge(options)
+    def initialize(initial_options)
+      @options = default_options
+      @options = @options.merge(initial_options)
+      @options = @options.merge(BrokenRecord::Config.slack_options)
     end
 
     def send!(message)
+      return unless @options[:summary]
```

Comment:
I'm going to create a separate PR to create some tests for this stuff (e.g., if someone removes the `summary` default option, it will stop sending our slack messages, so we want to force people to be thoughtful about removing default options)